### PR TITLE
chore: upgrade Node version

### PR DESCRIPTION
**Pull request by mcfdez**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v10.16.0
+v20.11.0
```
Comment (AlbertHernandez) [inline .nvmrc:1]: **issue (blocking):** we also should change the node version in the Dockerfile: https://github.com/AlbertHernandez/branch-name-action/blob/106d820c9f9275a7d0f8cf3d4ef637a46bb7b4ba/Dockerfile#L1

**Review by AlbertHernandez — CHANGES_REQUESTED** [review]

**Review by AlbertHernandez — COMMENTED** [review]
Let's also verify that the dependencies this project is using are compatible with the new version, I know in a lot of cases some dependencies in major versions drop compatibility with old node versions, so maybe we should upgrade them too

File: .nvmrc (hunk 1 above)
Comment (mcfdez) [reply .nvmrc:1]: Changed

**Comment by mcfdez** [comment]
> Let's also verify that the dependencies this project is using are compatible with the new version, I know in a lot of cases some dependencies in major versions drop compatibility with old node versions, so maybe we should upgrade them too

To test the action locally, do you use any specific way to do it? I know there is "Act", but is there any other way?

**Comment by mcfdez** [comment]
> Let's also verify that the dependencies this project is using are compatible with the new version, I know in a lot of cases some dependencies in major versions drop compatibility with old node versions, so maybe we should upgrade them too

I would take the opportunity to increase the dependencies, since in the PR of the refactor, we will make use of them.

**Comment by AlbertHernandez** [comment]
> > Let's also verify that the dependencies this project is using are compatible with the new version, I know in a lot of cases some dependencies in major versions drop compatibility with old node versions, so maybe we should upgrade them too
> 
> To test the action locally, do you use any specific way to do it? I know there is "Act", but is there any other way?

Usually, I test the actions in the own repository, pointing to the current code, I think we can do this by pointing to the branch name we are working in the tag, can you test it this way is working? Before continuing with your changes we should find some way to test all of this is working fine. Let me know if you have some issue and I can investigate about it

**Comment by mcfdez** [comment]
> > > Let's also verify that the dependencies this project is using are compatible with the new version, I know in a lot of cases some dependencies in major versions drop compatibility with old node versions, so maybe we should upgrade them too
> > 
> > 
> > To test the action locally, do you use any specific way to do it? I know there is "Act", but is there any other way?
> 
> Usually, I test the actions in the own repository, pointing to the current code, I think we can do this by pointing to the branch name we are working in the tag, can you test it this way is working? Before continuing with your changes we should find some way to test all of this is working fine. Let me know if you have some issue and I can investigate about it

I have tried to modify the testing workflow, but it asks for an approval from the mantainer.

image
image

**Pull request closed without merging**